CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 Changelog
 =========
 
-[v2.19](https://github.com/rigetti/pyquil/compare/v2.18.0...master) (in development)
```

Comment:
Do you want to add the new heading?

Reply:
It looks like it's usually done in a follow-on commit, along with updating the `..master` to `..tag` once the tag is created.

Reply:
Sounds like a way to game the github commit stats!

Reply:
Just following standard procedure!